website/components/openapi-page/index.jsx

```diff
@@ -54,7 +54,9 @@ function OpenApiPage({
               <h1 className={`${styles.categoryHeading} g-type-display-4`}>
                 {operationCategory.name}
               </h1>
-              <div ref={operationsRef}>
+              {/* @TODO: remove g-type-body-long className once https://github.com/hashicorp/react-components/pull/280 lands,
+              and has been pulled into this project. Until then, this className is needed to correct a typo in the g-content component. */}
+              <div className="g-type-long-body" ref={operationsRef}>
```

Comment:
Note: this is a really annoying thing I wish I caught earlier. However, it has very limited effect, as on all other docs pages (apart from these generated OpenAPI ones), we have our typography plugin in place, so correcting the className issue isn't critical (as most if not all content appears with the right type style anyways).

I have a fix in https://github.com/hashicorp/react-components/pull/280, which I'm going to merge tomorrow AM, and will likely be able to resolve this issue and close this out before merging this PR.

Reply:
Fixed now 😅